.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+branch = True
```

Comment:
Not sure I particularly care about branch coverage at this point in time. I'd be happy to drop it if it meant we didn't need an extra source file in the root of the repo.